assets/js/modules/analytics-4/components/dashboard/DashboardAllTrafficWidgetGA4/UserDimensionsPieChart.js

```diff
@@ -171,9 +171,10 @@ export default function UserDimensionsPieChart( props ) {
 		maxSlices: 5,
 		withOthers: true,
 		tooltipCallback: ( row, previousDateRangeRow, rowData ) => {
+			const rowValue = row?.metricValues?.[ 0 ]?.value || 0;
 			const difference =
-				previousDateRangeRow.metricValues?.[ 0 ].value > 0
-					? ( row.metricValues?.[ 0 ].value * 100 ) /
+				previousDateRangeRow?.metricValues?.[ 0 ].value > 0
```

Comment:
The last part `.value` should also be `?.` because `[ 0 ]` might be missing in the array and return `undefined`.

```suggestion
				previousDateRangeRow?.metricValues?.[ 0 ]?.value > 0
```